res/skins/Deere/style.qss

```diff
@@ -2190,3 +2201,28 @@ WEffectSelector {
 #CueDeleteButton:hover {
   background-color: #4B4B4B;
 }
+
+/* MainMenu */
+
+#MainMenu QMenu::item {
+  padding: 5px 13px 5px 35px;
+  margin: 0px 4px 0px 2px;
+}
+
+#MainMenu QMenu::item:checked {
+  padding: 5px 13px 5px 10px;
+}
+
+#MainMenu QMenu::item:unchecked {
+  padding: 5px 13px 5px 10px;
+}
+
+#MainMenu QMenu::indicator:checked,
+#MainMenu QMenu::indicator:checked:selected {
+  image: url(skin:/icon/ic_library_checkmark.svg);
+}
+
+#MainMenu QMenu::indicator:unchecked {
+  background: none;
+  border: 1px solid rgba(151,151,151,128);
+}
```

Comment:
Can be set together with `#LibraryContainer QTableView::indicator:unchecked` in line 276